src/main/java/nl/tudelft/contextproject/gui/DirectorLiveController.java

```diff
@@ -200,7 +212,11 @@ private void initializeButtons() {
         
         btnNext.setOnAction((event) -> {
             if (!endReached) {
-                script.next();
+                if (automaticCheck.isSelected()) {
```

Comment:
Use `script.next(automaticCheck.isSelected())` here instead of this if-statement.